test/models/driver_test.rb

```diff
@@ -60,22 +60,68 @@
 
   # Tests for methods you create should go here
   describe "custom methods" do
+    before do
+      @new_driver = Driver.create(name: "Mike", vin: "123", available: true)
+      @new_passenger = Passenger.create(name: "Mike", phone_num: "111-111-1211")
+
+    end
+    let(:trip_1) {
+      Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: Date.today, rating: 5, cost: 1000)
+    }
+    let(:trip_2) {
+      Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: Date.today, rating: 3, cost: 6334)
+    }
+    let(:trip_3){
+      Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: Date.today, rating: 3, cost: 1234)
+    }
     describe "average rating" do
-      # Your code here
+      it "returns average rating" do
+        trip_1
+        trip_2
+        trip_3
```

Comment:
I totally get what you all are trying to do with `trip_1`, `trip_2`, and `trip_3`-- the logic makes complete sense to me.

However, this code ends up reading weirdly and feeling awkward-- a single line that says `trip_1` confuses me as another programmer. `trip_1` sounds like a "thing", and "things" usually don't do anything on a line by itself... but truly, it's actually running `Trip.create()`!

It may make sense to not use the `let` block, and just move the `Trip.create` calls into the `before`, or even into this test itself.

```suggestion
        Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: Date.today, rating: 5, cost: 1000)
        Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: Date.today, rating: 3, cost: 6334)
        Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: Date.today, rating: 3, cost: 1234)
```

Also, now that we've learned about test fixtures, test fixtures might be an excellent option, too! We could have several fixtures for a driver with 0 trips, 1 trip, 2 trips, and 3 trips.